src/engine/renderer/tr_shade.cpp

```diff
@@ -964,35 +969,21 @@ static void Render_lightMapping( int stage )
 		{
 			GLimp_LogComment( "cubeProbeNearest && cubeProbeSecondNearest == NULL\n" );
 
-			// bind u_EnvironmentMap0
-			GL_BindToTMU( BIND_ENVIRONMENTMAP0, tr.whiteCubeImage );
-
-			// bind u_EnvironmentMap1
-			GL_BindToTMU( BIND_ENVIRONMENTMAP1, tr.whiteCubeImage );
+			cubeMap0 = tr.whiteCubeImage;
+			cubeMap1 = tr.whiteCubeImage;
 		}
 		else if ( cubeProbeNearest == nullptr )
 		{
 			GLimp_LogComment( "cubeProbeNearest == NULL\n" );
 
-			// bind u_EnvironmentMap0
-			GL_BindToTMU( BIND_ENVIRONMENTMAP0, cubeProbeSecondNearest->cubemap );
-
-			// u_EnvironmentInterpolation
-			gl_lightMappingShader->SetUniform_EnvironmentInterpolation( 0.0 );
+			cubeMap0 = cubeProbeSecondNearest->cubemap;
 		}
 		else if ( cubeProbeSecondNearest == nullptr )
 		{
 			GLimp_LogComment( "cubeProbeSecondNearest == NULL\n" );
 
-			// bind u_EnvironmentMap0
-			GL_BindToTMU( BIND_ENVIRONMENTMAP0, cubeProbeNearest->cubemap );
-
-			// bind u_EnvironmentMap1
-			// GL_SelectTexture( BIND_ENVIRONMENTMAP1 );
-			// GL_Bind( cubeProbeNearest->cubemap );
-
-			// u_EnvironmentInterpolation
-			gl_lightMappingShader->SetUniform_EnvironmentInterpolation( 0.0 );
+			cubeMap0 = cubeProbeNearest->cubemap;
+			// cubeMap1 = cubeProbeNearest->cubemap;
```

Comment:
remove this line?

Reply:
I prefer to not delete information from a code that does not work, we never attempted to debug, and we will have to debug one day. That would not be the same if this was working.

Reply:
But it's just noise here. It's not difficult to see what the code is trying to do - interpolate between two images if two of them are found, or if only one is found, just use the one image. Moreover, the case where the first one is missing and the case where the 2nd is missing should be the same.

Reply:
okok